components/DropDown.tsx

```diff
@@ -86,8 +86,35 @@ type Props = {
   className: string
 }
 
+export function getSortedMunicipalities(municipalitiesName: Array<string>) {
+  return municipalitiesName.sort((a, b) => a.localeCompare(b, 'sv'))
```

Comment:
I realized just now that this probably works on most of your machines without `'sv'` since you run Swedish, but wouldn't be surprised if it failed in CI/CD

Reply:
Exactly - this can't be guaranteed in CI.